src/reducer.js

```diff
@@ -118,6 +128,20 @@ export const actionReducers = {
   },
 
   /**
+   * Update the bunsen model
+   * @param {State} state - state to update
+   * @param {Action} action - action
+   * @returns {State} - updated state
+   */
+  [CHANGE_VIEW]: function (state, action) {
```

Comment:
Huh, this is an odd naming convention for a function. What's up with this? why not `changeView` ?